bin/run_tests

```diff
@@ -24,8 +24,37 @@ PROJECT_DIR=$(cd $(dirname $(dirname $0)) && pwd)
 # Make sawtooth scripts accessible
```

Comment:
This comment seems obsolete.  I think it should say "Make Avalon" not "Make sawtooth"

Reply:
changed to avalon.